.github/workflows/openshift-nightly.yml

```diff
@@ -50,3 +50,14 @@ jobs:
           touch .env
           ./demo/run-osm-demo.sh
           go run ./ci/cmd/maestro.go
+      - name: Cleanup resources
+        env:
+          BOOKWAREHOUSE_NAMESPACE: bookwarehouse
+          BOOKBUYER_NAMESPACE: bookbuyer
+          BOOKSTORE_NAMESPACE: bookstore
+          BOOKTHIEF_NAMESPACE: bookthief
+          MESH_NAME: osm
+          K8S_NAMESPACE: osm-system
+          TIMEOUT: 90s
```

Comment:
Does `TIMEOUT` need to be set for this script? I don't see any reference to it.

Reply:
TIMEOUT is used in the clean-kubernetes script in the `kubectl delete` commands: 

https://github.com/openservicemesh/osm/blob/main/demo/clean-kubernetes.sh#L15-L16